src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api.UnitTests/Controllers/CohortControllerTests/CreateEmptyCohortTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using AutoFixture;
+using FluentAssertions;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.CommitmentsV2.Api.Controllers;
+using SFA.DAS.CommitmentsV2.Api.Types.Requests;
+using SFA.DAS.CommitmentsV2.Api.Types.Responses;
+using SFA.DAS.CommitmentsV2.Application.Commands.AddCohort;
+using SFA.DAS.Testing;
+
+namespace SFA.DAS.CommitmentsV2.Api.UnitTests.Controllers.CohortControllerTests
+{
+    [TestFixture]
+    [Parallelizable]
+    public class CreateEmptyCohortTests : FluentTest<CreateEmptyCohortTestsFixture>
+    {
+        [Test]
+        public async Task WhenPostRequestReceived_ThenShouldReturnResponse()
+        {
+            await TestAsync(
+                f => f.Create(),
+                (f, r) => r.Should().NotBeNull()
+                    .And.BeOfType<OkObjectResult>()
+                    .Which.Value.Should().NotBeNull()
+                    .And.Match<CreateCohortResponse>(v =>
+                        v.CohortId == f.Result.Id &&
+                        v.CohortReference == f.Result.Reference));
+        }
```

Comment:
FYI we are trying not to to use the TestAsync() testing methods when creating a new test class. But I can see why you did, as the other tests in this area use this pattern.